base/applications/rapps/appview.cpp

```diff
@@ -1410,7 +1252,7 @@ BOOL CAppsListView::SetDisplayAppType(APPLICATION_VIEW_TYPE AppType)
         SetCheckboxesVisible(TRUE);
         break;
 
-    case AppViewTypeEmpty:
+    //case AppViewTypeEmpty:
     default:
         break;
```

Comment:
what is the reasoning behind keeping that commented line?

Reply:
forgot to remove it.